worker-jobtracking/src/test/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerFactoryTest.java

```diff
@@ -115,13 +123,14 @@ public void testTrackingReportTask() throws Exception
         final JobTrackingWorkerFactory workerFactory = createJobTrackingWorkerFactory(codec, reporter);
 
         //Test
+        workerFactory.getWorker(wtd).doWork();
```

Comment:
I don't understand what we've achieving here by doing this twice.

Reply:
Restored